addons/sales_team/models/crm_team.py

```diff
@@ -22,24 +22,26 @@ class CrmTeam(models.Model):
     @api.model
     @api.returns('self', lambda value: value.id if value else False)
     def _get_default_team_id(self, user_id=None):
-        if not user_id:
-            user_id = self.env.uid
-        company_id = self.sudo(user_id).env.user.company_id.id
-        team_id = self.env['crm.team'].sudo().search([
-            '|', ('user_id', '=', user_id), ('member_ids', '=', user_id),
-            '|', ('company_id', '=', False), ('company_id', 'child_of', [company_id])
-        ], limit=1)
-        if not team_id and 'default_team_id' in self.env.context:
+        if 'default_team_id' in self.env.context:
```

Comment:
This method is highly controversial and I would like not changing it. Why did you change its behavior ?

According to spec you have some default teams (pos / website) -> those should be trigerred only if the current user is not member or manager of a team. Own team > other default values.